vllm/model_executor/layers/quantization/kernels/scaled_mm/aiter.py

```diff
@@ -20,25 +27,20 @@ def get_min_capability(cls) -> int:
     @classmethod
     def can_implement(
             cls, c: ScaledMMLinearLayerConfig) -> Tuple[bool, Optional[str]]:
-        if current_platform.is_cpu():
+        if current_platform.is_cpu() or not current_platform.is_rocm():
```

Comment:
Why can't this be simpler?
```suggestion
        if not current_platform.is_rocm():
```

Reply:
I got mixed up as ROCm stack also works on CPU. Ok. I have removed the check for CPU.